oat-sa/extension-pcisample
# Hotfix/INV-707/2023.05/Broken tooltip

### Related to: https://oat-sa.atlassian.net/browse/INV-707

Backport of https://github.com/oat-sa/extension-pcisample/pull/144

### Summary

Restore the tooltip helper in the textReader PCI.

![image](https://github.com/oat-sa/extension-pcisample/assets/1500098/97fd53c9-3584-498f-b437-e7b925f3ccad)
![image](https://github.com/oat-sa/extension-pcisample/assets/1500098/c6c32a6f-8e29-4ea9-932b-163e489c3975)

Also:
- make sure it works with TAO Advance.
- fix the tooltip not showing up in the item authoring preview.

### Details

When migrating the PCI to the IMS standard, we made some surgery to remove or adapt incompatible code. However, it was a tad too aggressive, and [the helper responsible for preparing the tooltip was removed together with the one that brought MathJAX](https://github.com/oat-sa/extension-pcisample/pull/87/files#diff-82c38155dd7370921087076be131fa5e089961d18643a4bdb36253e7ad5d3470L186).

### How to test
- install TAO with the extension pciSamples from tao-community v2023.05.4
- check out the branch
- update TAO
- have an item with the textReader PCI and some tooltips (you can use the ones attached to the ticket)
- publish a delivery (or pick up an existing one if any
- take the test and check the tooltip works
